tox.ini

```diff
@@ -141,9 +141,11 @@ commands =
 description = Prepare Python source and binary packages for release.
 basepython = python3
 skip_install = false
+extras =
+    dev
 commands =
     bash -c 'rm -rf build/* dist/*'
-    python setup.py sdist bdist_wheel
+    python -m build
```

Comment:
This is the new basic build backend that replaced running `setup.py`. It creates a source + wheel distribution by default.